Dockerfile.integration-test

```
@@ -0,0 +1,14 @@
+FROM bats/bats:latest
+
+ARG KUBECTL_VERSION=v1.29.2
```

In greenhouse we use 1.30.2 version. would be nice to keep the versions aligned everywhere
```suggestion
ARG KUBECTL_VERSION=v1.30.2
```

---

I have fixed in https://github.com/cloudoperators/greenhouse-extensions/pull/237/commits/924cf43397727fe509b9c81e936192123c6d484f commit.